<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyImages</class>
 <widget class="QDialog" name="MyImages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="push_back">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>481</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Images:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>481</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_view">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Images and its Viewers</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
